crates/typst-library/src/math/style.rs

```diff
@@ -88,12 +106,12 @@ pub fn sans(
 ///
 /// (The box is not conceptually necessary, but unfortunately currently needed
 /// due to limitations in Typst's text style handling in math.)
-#[func(title = "Calligraphic", keywords = ["mathcal", "mathscr"])]
-pub fn cal(
+#[func(title = "Script", keywords = ["mathscr", "roundhand"])]
```

Comment:
```suggestion
#[func(title = "Script Style", keywords = ["mathscr", "roundhand"])]
```

This is probably better for clarity (`math.script` is "Script Size").

Reply:
Oh, I completely overlooked that

Reply:
If `math.script` is Script Size already is adding "Style" really needed then?

Reply:
There are already many cases where searching for a function in the reference yields multiple results with identical or similar names. I think we should try to disambiguate those as much as possible.